packages/expo-updates/android/src/main/java/expo/modules/updates/UpdatesConfiguration.kt

```diff
@@ -18,26 +18,26 @@ class UpdatesConfiguration private constructor (
   val checkOnLaunch: CheckAutomaticallyConfiguration,
   val hasEmbeddedUpdate: Boolean, // used only for expo-updates development
   val requestHeaders: Map<String, String>,
-  private val codeSigningCertificate: String?,
-  private val codeSigningMetadata: Map<String, String>?,
+  val codeSigningCertificate: String?,
+  val codeSigningMetadata: Map<String, String>?,
 ) {
   enum class CheckAutomaticallyConfiguration {
     NEVER, ERROR_RECOVERY_ONLY, WIFI_ONLY, ALWAYS
   }
 
-  constructor(context: Context?, map: Map<String, Any>?) : this(
-    isEnabled = map?.readValueCheckingType<Boolean>(UPDATES_CONFIGURATION_ENABLED_KEY) ?: context?.getMetadataValue("expo.modules.updates.ENABLED") ?: true,
-    expectsSignedManifest = map?.readValueCheckingType(UPDATES_CONFIGURATION_EXPECTS_EXPO_SIGNED_MANIFEST) ?: false,
+  constructor(context: Context?, overrideMap: Map<String, Any>?) : this(
```

Comment:
Worth making overrideMap optional? Or do you think the explicit null is better?

Reply:
Explicit null looks better in intelli-j than it does here in github since the IDE adds a label UI element in front of it.

That being said, I had it be optional for a bit, but then made it non-optional once I realized context could also be null.